src/lib/resource/sections/mainapplicant-enter-your-name.js

```diff
@@ -4,47 +4,90 @@ module.exports = {
     section: {
         schema: {
             $schema: 'http://json-schema.org/draft-07/schema#',
-            title: 'Enter your name',
             type: 'object',
-            required: [
-                'q-mainapplicant-title',
-                'q-mainapplicant-first-name',
-                'q-mainapplicant-last-name'
-            ],
-            additionalProperties: false,
-            properties: {
-                'q-mainapplicant-title': {
-                    title: 'Title',
-                    type: 'string',
-                    maxLength: 6,
-                    errorMessage: {
-                        maxLength: 'Title must be 6 characters or less'
-                    }
-                },
-                'q-mainapplicant-first-name': {
-                    title: 'First name',
-                    type: 'string',
-                    maxLength: 70,
-                    errorMessage: {
-                        maxLength: 'First name must be 70 characters or less'
-                    }
-                },
-                'q-mainapplicant-last-name': {
-                    title: 'Last name',
-                    type: 'string',
-                    maxLength: 70,
+            allOf: [
+                {
+                    title: 'Enter your name',
+                    meta: {
+                        compositeId: 'mainapplicant-name',
+                        classifications: {
+                            theme: 'main-applicant-details'
```

Comment:
should the theme be consistent with the ids/compositeIds and have no hyphen between `main` and `applicant`? More a question than a required change

Reply:
they probably should

Reply:
And the themes are distinct from p and q ids